lib/hanami/assets/bundler.rb

```diff
@@ -116,8 +128,11 @@ def generate_manifest
 
       # @since 0.1.0
       # @api private
-      def store_manifest(asset, target)
-        @manifest[_convert_to_url(::File.expand_path(asset))] = _convert_to_url(::File.expand_path(target))
+      def store_manifest(asset, target, sri_value)
+        @manifest[_convert_to_url(::File.expand_path(asset))] = {
+          target: _convert_to_url(::File.expand_path(target)),
```

Comment:
@accuser I've seen that the configuration can handle multiple hashing algorithms, but the manifest reports only one `"sri"` value. Shall we change and make the value an array?

```json
{"/assets/application.js":{"target":"/assets/application-123abc.js","sri":["sha256-def456"]}}
```